script.js

```diff
@@ -62,7 +62,7 @@ taskForm.onsubmit = async (e) => {
       dinero: lossRateDinero,
       neelam: lossRateNeelam,
     },
-    isNoteworthy: isNoteworthy || false,
+    isNoteworthy: isNoteworthy = (isNoteworthy == 'true'),
```

Comment:
Hi @josedemouramachado , **isNoteworthy** returns "on" if it's checked and "undefined" if it's not checked. Please do changes according to it.

Reply:
Hello @aman7870,

My bad. Changed to isNotworthy = (isNoteworthy == 'on')